platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/GossipEvent.java

```diff
@@ -202,11 +202,23 @@ public void buildDescriptor() {
         this.descriptor = hashedData.createEventDescriptor();
     }
 
+    /**
+     * {@inheritDoc}
+     */
     @Override
     public long getGeneration() {
         return hashedData.getGeneration();
     }
 
+    /**
+     * Get the birth round of the event.
+     *
+     * @return the birth round of the event
+     */
+    public long getBirthRound() {
```

Comment:
My personal preference is to not include these kind of data pass-throughs, as it can complicate making mocks of the objects. Up to you whether to keep it or not, though

Reply:
I agree with this sentiment in general. Method removed.